### DISCUSSION TITLE
Question 46: Searching for unattached multimedia [WCG]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#592] [Q&A] [answered]
Description: Is there a straightforward way to search for multimedia records that aren’t attached in eCatalogue? I can think of roundabout ways that involve running reports but there might be an easier way.

**Comment by rondlg** [comment]
Response: 
If you’re looking for MM in a particular area, try this search criteria:
(Audubon Core tab) Associated Specimen = \\!\\*
Might want to include Associated Observation = \\!\\*
(Security tab) Department = Botany 
	…Many records might be attached directly to Taxonomy 

Response-response [WCG] multimedia irn 1051602 is attached to catalogue irn 3586341 but has nothing in Associated Specimen or Associated Observation in audubon core tab

**Action for IT**: Requested a back-end refresh of Audubon Core fields for ALL mm records from Axiell
- MM irn 1659443 – tried refreshing on front end (re-attach to Catalog + edit/refresh/resave MM), but still not populating AudCore fields
- Update [2-Jun]: HORNBILL ticket submitted - https://customer.hornbill.com/axiell/servicemanager/request/view/IN00137700/

**Comment by magpiedin** [comment, marked as answer]
If the refresh takes a while, here's one other workaround:

- *on Audubon Core tab*
  - Taxon Coverage (AudTaxonCoverage) = \\!\\*
  - Related Geography (AudRelatedGeography) = \\!\\*
- *on Security tab*
  - Department = Botany 

(Not sure why, but the Tax & Sites values seem to come through ok for many attached MM records mistakenly missing their 'Assoc. Specimen' read-through. The above search should filter those out.)

**Comment by magpiedin** [comment]
Update: If a Catalogue record doesn't have an attached Collection Event, the Associated Specimen (& Associated Observation) fields won't populate.  (e.g. for MM irn 1659443, the reverse-attached Catalogue irn 199663 doesn't have a Col.Event)
